cvat/apps/iam/serializers.py

```diff
@@ -23,6 +28,20 @@
 class RegisterSerializerEx(RegisterSerializer):
     first_name = serializers.CharField(required=False)
     last_name = serializers.CharField(required=False)
+    email_verification_required = serializers.SerializerMethodField()
+    key = serializers.SerializerMethodField()
+
+    @extend_schema_field(serializers.BooleanField)
+    def get_email_verification_required(self, obj: Union[Dict, User]) -> bool:
+        return allauth_settings.EMAIL_VERIFICATION == allauth_settings.EmailVerificationMethod.MANDATORY
+
+    @extend_schema_field(serializers.CharField(allow_null=True))
+    def get_key(self, obj: Union[Dict, User]) -> Optional[str]:
```

Comment:
Why is this the type of `obj`? How can it be a dict?

Reply:
https://github.com/iMerica/dj-rest-auth/blob/6807ea06a9ce0f4f11d607d0005f5d99aabe5f99/dj_rest_auth/registration/views.py#L62

Reply:
I see.